packages/manager/modules/emailpro/src/domain/dkim-autoconfig/emailpro-domain-dkim-autoconfig.html

```diff
@@ -3,35 +3,22 @@
         data-ng-if="!ctrl.dkimForNoOvhCloud"
         data-wizard
         data-wizard-on-cancel="resetAction"
-        data-wizard-on-finish="ctrl.configDkim.bind(ctrl)"
+        data-wizard-on-finish="ctrl.onFinishDkim.bind(ctrl)"
+        data-wizard-confirm-button-text="ctrl.dkimStatus === ctrl.GLOBAL_DKIM_STATUS.OK ?
+        'emailpro_tab_domain_diagnostic_dkim_deactivate' : 'emailpro_tab_domain_diagnostic_dkim_activate' | translate"
         data-wizard-hide-confirm-button="ctrl.hideConfirmButton"
-        data-wizard-title="ctrl.dkimStatus === ctrl.GLOBAL_DKIM_STATUS.NOK ?
+        data-wizard-title="ctrl.dkimStatus === ctrl.GLOBAL_DKIM_STATUS.NOK ||
+                           ctrl.dkimStatus === ctrl.GLOBAL_DKIM_STATUS.OK ?
```

Comment:
Hello, not sure what the code does here, but is ok that we check if status is "OK OR NOK", does seem two opposite values to me, but you return the same. Is it intended?

Reply:
> Hello, not sure what the code does here, but is ok that we check if status is "OK OR NOK", does seem two opposite values to me, but you return the same. Is it intended?

Yes, because there are many other possible status, this is what i want in NOK and OK case